CONTRIBUTING.md

```diff
@@ -0,0 +1,108 @@
+# Contributing {#about_contributing}
+
+First of all, thanks for wanting to contribute to SeqAn! Community is important to us and we strive to maintain a great
+culture and atmosphere. Please have a look at our [Code of Conduct](\ref about_code_of_conduct).
+
+The following is a guide that helps stream-line the process of adding changes. If you haven't contributed to SeqAn
+before, don't worry about getting something wrong, that's absolutely OK. However, following this guide closely
+will reduce work for all of us and increases the chance of your changes being merged quickly.
+
+# Pull request workflow
+
+The overall workflow of contributing changes to this repository is:
+
+  1. Create an issue with bug report or feature request.
+  2. Wait for comments from the team and then collectively decide on the best way to solve the issue.
+  3. Fork the repository and create a branch on your fork.
+  4. Commit the solution to your local branch.
+  5. Create a pull-request to our master branch from your branch.
```

Comment:
we should name this already `develop` branch, or discuss how we want to proceed after the release.

Reply:
I thought we already discussed this, but maybe we didn't. I am against the develop branch :fist_raised:

Reply:
it was not, but we discussed it now 😉